php/PaypalIPN.php

```diff
@@ -136,11 +136,10 @@ public function verifyIPN()
 
         curl_close($ch);
 
-        // Check if PayPal verifies the IPN data, and if so, return true.
+        // Check if PayPal verifies the IPN data, and if so, return data array.
         if ($res == self::VALID) {
-            return true;
-        } else {
-            return false;
+            return $myPost;
         }
+        return false;
```

Comment:
It's best not to  return multiple types from a function -  it would make more sense to return null vs false here.